### ISSUE TITLE
Floating window - buggy

### ISSUE THREAD

**Issue by faced0a** [ianyh/Amethyst#1237]
Is there a way to force a windows to be always tiling? The notes.app is always floating, there's no extra config, I use default.
Another point: if I press Mod1-t in an already floating window, it should turn into tiling, right? Why does it not work? Is it a known bug or a missing feature?

**Comment by ianyh** [comment]
If you also can't force the window to tile it means Amethyst missed the whole application. Not sure how that would be happening so consistently, as I also use the Notes app with it tiling fine.

**Comment by faced0a** [comment]
It's really a strange unexpected behavior since it haven't added custom config, I use default one, do you have a tip to debug such situations? I will appreciate it.

**Comment by faced0a** [comment]
Found it happens only with Notes.app and Reminders.app, all others works as expected.

**Comment by faced0a** [comment]
@ianyh Ok, I figured out the issue, it happens due the fact the Reminders and Notes app have been flagged to be available on "All desktops"

<img width="280" alt="image" src="https://user-images.githubusercontent.com/94846799/167297654-b69f67f4-b6cc-4388-bc63-1936400c1778.png">

Maybe it can worth at least documentation this behavior somewhere, so others won't spend time on it. Dunno if it can be considered a bug or missing feature though. Maybe We can consider "All Desktops" apps to be tiling everywhere at all.

**Comment by ianyh** [comment]
Yeah, it would be good for a case on a troubleshooting page. This has come up before.
